manifests/s/sunfish-shogi/shogihome/1.19.4/sunfish-shogi.shogihome.locale.ja-JP.yaml

```
@@ -0,0 +1,30 @@
+# Created with komac v2.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: sunfish-shogi.shogihome
+PackageVersion: 1.19.4
+PackageLocale: ja-JP
+Publisher: sunfish-shogi
+PublisherUrl: https://github.com/sunfish-shogi
+PublisherSupportUrl: https://github.com/sunfish-shogi/shogihome/issues
+PackageName: ShogiHome
+PackageUrl: https://github.com/sunfish-shogi/shogihome
+License: MIT
+LicenseUrl: https://github.com/sunfish-shogi/shogihome/blob/v1.19.4/LICENSE
+Copyright: Copyright (c) 2022 Kubo Ryosuke
+CopyrightUrl: https://github.com/sunfish-shogi/shogihome/blob/v1.19.4/LICENSE#L3C1-L3C32
+ShortDescription: About

```

Sorry for mis commit 🙇‍♂️ 
```suggestion
ShortDescription: 高機能な将棋の GUI
```